### ISSUE TITLE
Finding a user by email address not possible

### ISSUE THREAD

**Issue by rolfschmidt** [zammad/zammad#3190]
### Infos:

* Used Zammad version: 3.5 (current develop)
* Installation method (source, package, ..): package
* Operating system: Debian 9
* Database + version: postgreSQL 9
* Elasticsearch version: 7.9.1
* Browser + version: Google Chrome 85.0.4183.83
* Note: double checked on a hosted instance as well. ES being available or not does not affect the behavior.

### Expected behavior:

When creating a deletion task, you can find the user by it's name or email address / login. 
So basically like everywhere in Zammad.

### Actual behavior:

You can't find a to delete user by it's email address.

### Steps to reproduce the behavior:

* copy the email address of any existing user
* go to data privacy and click on "New Deletion Task"
* for user paste in your email address (you can also type it by hand, doesn't matter)

![grafik](https://user-images.githubusercontent.com/5445062/92878735-b283fa00-f40c-11ea-9de0-c4925b23d0dc.png)

Issue by @MrGeneration

**Comment by Mebus** [comment]
I can confirm this problem. Are there any workarounds?

**Comment by MrGeneration** [comment]
No workaround, it's technically not possible at the moment.

**Comment by Mebus** [comment]
Not even directly in the database?

**Comment by thorsteneckel** [comment]
@Mebus - this is already implemented and currently in review. Unfortunately it came up too late and didn't make it into the release. However, it will be added over the next days.
The workaround is to search users by name.

**Comment by rolfschmidt** [comment]
Hi @Mebus ,
can't you search for the email address in the users admin interface?

![grafik](https://user-images.githubusercontent.com/5445062/93893400-e0f7b400-fced-11ea-9407-2c6133ac2a9c.png)

The bug should be fixed soon with 3.5.1.

**Comment by rolfschmidt** [comment]
or global search

![grafik](https://user-images.githubusercontent.com/5445062/93893578-14d2d980-fcee-11ea-99c0-8b0d489566c6.png)

**Comment by Mebus** [comment]
I can search, but can't find him. I am running 3.5.0-1600773746.911896f5.stretch at the moment.

Mebus

**Comment by rolfschmidt** [comment]
@Mebus In the users admin interface you need to be sure that all roles are checkmarked. The global search should display him too hmm...

Alternatively you could try to search for a ticket of your customer and delete him from there?

![grafik](https://user-images.githubusercontent.com/5445062/93894788-692a8900-fcef-11ea-9841-b92440d7089d.png)

**Comment by MrGeneration** [comment]
Sorry but this seems to turn into a technical question.
If you can't find an existing user via the search function, please ensure that your searchindex is up to date.

If this still doesn't help, please consult the community for technical questions: https://community.zammad.org

**Comment by thorsteneckel** [comment]
Fixes with the commit above for develop / 3.6. Backport to stable / 3.5 ready in about 2h.
